### DISCUSSION TITLE
Hardware button hold - default action, different entity

### DISCUSSION THREAD

**Discussion by kkopachev** [Blackymas/NSPanel_HA_Blueprint#2340] [Q&A]
I have a configuration where I'd like physical button to control relay on short press, however want to control another light on long press (hold). 
Basically I have 2 lights, one is wired to relay, another one is zigbee controlled through HA. 
If I set the button to control zigbee light, then default action on long press would bring up a page with more controls of such light: brightness, color, etc. 
However, when using relay for short press, there seem to be no way to call "default action" on another entity. I can only set custom action light "Toggle light" for zigbee entity.
What I want to get is short press controlling relay, hold showing light control page for another entity.
Is that possible somehow? I dug through all available actions, and they seem to be very basic, no way to show full page of all control options

**Comment by koswatana** [comment]
## 意思決定が必要な論点

### 1. メタデータ形式

どのプロファイルを採用するか。

- **DCAT-US 1.1**：現行（ArcGIS）との互換性あり。ジオコーダーはすでに対応済み。
- **DCAT-GoJ**：デジタル庁推奨（[DS-469 メタデータ導入実践ガイドブック](https://www.digital.go.jp/assets/contents/node/basic_page/field_ref_resources/890e5d96-d63c-4b77-bd3e-cc89487393e3/1ead1a75/data_strategy_gif_469_guidebook_metadata.docx)）。W3C DCAT 2.0 ベースの日本行政標準。BregDCAT（ベース・レジストリ向け）参考。

### 2. エントリ粒度

#43 の分割単位の決定に依存。#43 が先に決まれば自動的に決まる。

### 3. 配信方法

どこに・どのように配置するか。

- さくらのオブジェクトストレージに JSON ファイルを配置
- 専用エンドポイントを立てる

### 4. ジオコーダー（abrg）の移行

現行 abrg は ArcGIS DCAT を参照している。形式・URL が変わる場合、abrg 側の改修タイミングをどうするか。
